website/docs/jellyfish/api/poolpair.md

```diff
@@ -161,6 +161,29 @@ interface PoolShareOptions {
 }
 ```
 
+## removePoolLiquidity
+
+Remove pool liquidity transaction
+
+```ts title="client.poolpair.addPoolLiquidity()"
+interface poolpair {
+  removePoolLiquidity (from: removePoolLiquiditySource, shareAddress: string, options: removePoolLiquidityOptions = {}): Promise<string>
```

Comment:
removePoolLiquiditySource 
removePoolLiquidityOptions

Isn't that the First letter should be capitalized?

Reply:
Yes, thanks for noticing! I'll make the changes.